src/plugins/intel_gpu/src/kernel_selector/cl_kernels/fully_connected_gpu_bf_tiled.cl

```diff
@@ -193,7 +193,28 @@ KERNEL(fc)(
                 ACCUMULATOR_TYPE* w = (ACCUMULATOR_TYPE*)(&wei);
                 unroll_for(uint kii = 0; kii < TILE_K; ++kii) {
                     unroll_for(uint fi = 0; fi < TILE_OFM; ++fi) {
-                        w[kii * TILE_OFM + fi] = (w[kii * TILE_OFM + fi] - dzp[fi]) * ds[fi];
+                        const uint w_idx = kii * TILE_OFM + fi;
+                        uint offset_ofm = out_f + fi*SIMD + get_sub_group_local_id();
```

Comment:
```suggestion
                        const uint offset_ofm = out_f + fi*SIMD + sglid ;
```
And maybe it makes sense to move `w_idx` and `offset_ofm` calculations outside TILE_OFM loop?

Reply:
Fixed in https://github.com/openvinotoolkit/openvino/pull/20491/commits/88932f73cd69f052f0e8937d8e898c253a9e2bac

> And maybe it makes sense to move w_idx and offset_ofm calculations outside TILE_OFM loop?

Those indices depend on `fi`. If you mean that we can move part of the offset compute outside of OFM loop, then looks like compiler does that.